Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - BitLocker Device Encryption Enabled.json

```diff
@@ -46,5 +46,5 @@
     "startupAuthenticationTpmPinUsage": "allowed",
     "startupAuthenticationTpmUsage": "allowed"
   },
-  "description": "Enables and configures BitLocker device encryption"
+  "description": "What is it: Enables and configures BitLocker device encryption for physical devices. \n Simeon Notes: Device encryption is essential to protecting data on physical devices. Note that BitLocker requires a machine to have TPM 1.2 or later, which excludes older hardware and virtual machines. \n Learn More: https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-device-encryption-overview-windows-10"
```

Comment:
**What does this do?**
Enables Remote Desktop access to the device for users that are members of the 'Remote Desktop Users' local group.

**Why should you use this?**
 If you want to allow Remote Desktop access to the device.

**What is the end-user impact?**
Users in the 'Remote Desktop Users' local group will be able to connect to remote devices. 

**Learn more**
[Tactical considerations for creating Windows deployment rings](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tactical-considerations-for-creating-windows-deployment-rings/ba-p/746979)